data/blank_mod/common/ideas/navy_spirits.txt

```diff
@@ -276,6 +404,7 @@ ideas = {
 		} 
 		bureau_of_ordnance_spirit = {
 			ledger = navy
+			visible = { tag = USA }
```

Comment:
Please either remove the country-specific visible or the whole idea

Reply:
Removed - it's actually a negative modifier. Meant for historical scenarios presumably.